content/docs/command-reference/params/index.md

```diff
@@ -143,6 +143,61 @@ $ dvc run -n train -d logs/ -o users.csv \
           python train.py
 ```
 
+## Examples: Python parameters file
+
+Consider this parameters file in Python format, named `params.py`:
+
+```python
+IS_BOOL: bool = True
+CONST = 5
+
+
+class TrainConfig:
+    EPOCHS = 70
+
+    def __init__(self):
+        self.layers = 9
+
+
+class TestConfig:
+    TEST_DIR = "path"
+    METRICS = ["metric"]
+```
+
+The following [stage](/doc/command-reference/run) depends on params
+`IS_BOOL`, `CONST`, `layers`, and `EPOCHS`:
+
+```dvc
+$ dvc run -n train -d users.csv -o model.pkl \
+          -p params.py:IS_BOOL,CONST,TrainConfig.EPOCHS,TrainConfig.layers \
+          python train.py
+```
+
+Resulting `dvc.yaml` file (notice the `params` list):
+
+```yaml
+stages:
+  train:
+    cmd: python train.py
+    deps:
+      - users.csv
+    params:
+      - IS_BOOL
+      - CONST
+      - TrainConfig.EPOCHS
+      - TrainConfig.layers
+    outs:
+      - model.pkl
+```
```

Comment:
Should we also/alternatively show the stage in dvc.lock though? Including the param values.

Reply:
Or at least make a note (md block quote) about that, perhaps.

Reply:
Added

Reply:
Thanks @aandrusenko